core/src/main/java/org/jboss/pnc/rex/core/counter/MaxConcurrentCounter.java

```diff
@@ -18,52 +18,92 @@
 package org.jboss.pnc.rex.core.counter;
 
 import io.quarkus.infinispan.client.Remote;
+import jakarta.annotation.Nullable;
 import jakarta.inject.Inject;
 import org.infinispan.client.hotrod.RemoteCache;
 import org.infinispan.client.hotrod.VersionedValue;
 
 import jakarta.enterprise.context.ApplicationScoped;
 import org.jboss.pnc.rex.core.config.ApplicationConfig.Options.TaskConfiguration;
 
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+
 @MaxConcurrent
 @ApplicationScoped
 public class MaxConcurrentCounter implements Counter {
 
     public static final String MAX_COUNTER_KEY = "MAX_CONCURRENT";
+    public static final String NAME_SEPARATOR = "-";
```

Comment:
Don't we want this separator (or all constants) in some constants class? I see name separator in 2 places.

Reply:
okay, will do